channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/TelephonyReactions.kt

```diff
@@ -108,6 +113,71 @@ class TelephonyReactions(private val bargeInDefaultProps: BargeInProperties) : J
         bargeIn = BargeInProperties(mode, trigger, noInterruptTimeMs)
     }
 
+    /**
+     * This method overrides the TTS configuration of the phone channel used for the current call.
+     *
+     * Example usage:
+     * ```
+     * state("tts") {
+     *    action {
+     *        reactions.telephony?.setTtsConfig(TtsConfigAimyvoice("Никита"))
+     *    }
+     * }
+     * ```
+     * The configuration is determined by the TtsProviderConfig subclass passed in as a parameter.
+     * The appropriate provider-specific configuration settings are extracted and assigned
+     * to the corresponding fields in the TtsConfig class.
+     * @param config TtsProviderConfig subclass containing the provider-specific configuration settings to use for TTS.
+     * */
+    fun <T : TtsProviderConfig> setTtsConfig(config: T) {
```

Comment:
Seems like generic is not required here, can just take `config: TtsProviderConfig` as a parameter

Reply:
Yes. Fixed it.